account_operating_unit/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# © 2016-17 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+{
+    "name": 'Accounting with Operating Units',
+    "summary": "Introduces Operating Unit fields in invoices and "
```

Comment:
For the sake of consistency with the other modules.
```suggestion
    "summary": "Introduces Operating Unit (OU) in invoices and "
```